macros/src/lib.rs

```diff
@@ -138,6 +140,52 @@ pub fn test_traced(attr: TokenStream, item: TokenStream) -> TokenStream {
     TokenStream::from(expanded)
 }
 
+/// Prefix a test name with a nextest filter group.
+///
+/// This renames `test_some_behavior` into `test_<group>_some_behavior`, making
```

Comment:
I don't think `test_` is always present ... is there a way to produce this filter using the expanded test macro instead 🤔